sourmash/sourmash_args.py

```diff
@@ -404,6 +411,21 @@ def _load_database(filename, traverse, traverse_yield_all):
         except:
             pass
```

Comment:
why is it better to pass here, then try again below (rather than try/except within the exception)?

Reply:
in this case, because we have so many things to try. if we did nested try/excepts it would end up indenting to the right.

Reply:
great, thanks!